app/src/main/java/com/example/fittracker/UserDao.java

```diff
@@ -0,0 +1,25 @@
+package com.example.fittracker;
+
+import android.arch.lifecycle.LiveData;
+import android.arch.persistence.room.Dao;
+import android.arch.persistence.room.Insert;
+import android.arch.persistence.room.Query;
+
+@Dao
+public interface UserDao {
+    @Insert
+    void insert(User user);
+
+    @Query("SELECT * FROM user_table " +
+            "WHERE mMail == :userEmail AND password == :userPassword")
+    public LiveData<User> fetchUserData(String userEmail, String userPassword);
+
+    @Query("SELECT COUNT(*) from user_table WHERE mMail == :userEmail")
+    public int fetchUserExist(String userEmail);
+
+    @Query("SELECT COUNT(*) from user_table WHERE mMail == :userEmail AND password == :userPassword")
+    public int fetchUserLogInValid(String userEmail, String userPassword);
+}
+
+
+// Utilizar POJO para consultas que devuelvan menos columnas.
```

Comment:
use English please and, if it not necessary remove it pls

Reply:
Done